emgui/templates/rest_framework/login_base.html

```diff
@@ -1,7 +1,59 @@
 {% extends "rest_framework/base.html" %}
+{% load rest_framework %}
 
 {% block content %}
-    {% include "rest_framework/login_form.html" %}
-{% endblock %}
+    <h3>MGnify login</h3>
+    <form action="{% url 'rest_framework:login' %}" role="form" method="post" class="vf-stack vf-stack--400">
+      <div class="vf-form__item vf-stack">
+
+        <label for="text" class="vf-form__label vf-form__label--required">
```

Comment:
Is this compatible with the current website styling?.

Reply:
Everything UI in this PR is VF2 style, so will look a bit out-of-place with the stable website, but okay with the beta website. So perhaps makes sense to release this to prod at the same time as making the frontend a public beta.